server/db/models/link.js

```diff
@@ -0,0 +1,23 @@
+const Sequelize = require('sequelize');
+const db = require('../db');
+
+module.exports = db.define('link', {
+  link: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  dateModified: {
+    type: Sequelize.DATE,
+  },
+  dateCreated: {
+    type: Sequelize.DATE,
+  },
```

Comment:
You need to disambiguate these columns from the automatic columns used by Sequelize, and make it more obvious that this information is about the link — not the db row.